### ISSUE TITLE
improve contrast of non-text component styling

### ISSUE THREAD

**Issue by anandamaryon1** [nhsuk/nhsuk-service-manual-community-backlog#508]
## What
We've received the following feedback from an accessibility audit (https://github.com/nhsuk/nhsuk-frontend/issues/927):

> # Bug Report
> ## What is the issue?
> An accessibility audit of our site has flagged the following non-compliance with WCAG Level AA: 1.4.11 Non-text Contrast. The contrast ratio of the dashes / bullet points in a contents list (https://service-manual.nhs.uk/design-system/components/contents-list) is 1.83:1
> 
> **WCAG Success Criterion** [Success Criterion 1.4.11 Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html): The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s): User Interface Components: Visual information required to identify user interface components and states, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author; Graphical Objects: Parts of graphics required to understand the content, except when a particular presentation of graphics is essential to the information being conveyed.
> 
> **WCAG Sufficient technique** [G207](https://www.w3.org/WAI/WCAG21/Techniques/general/G207): Ensuring that a contrast ratio of 3:1 is provided for icons
> 
> ## What steps are required to reproduce the issue?
> Go to any site that contents lists, e.g.: https://service-manual.nhs.uk/design-system/components/contents-list
> 
> ## What was the environment where this issue occurred?
> Can be observed on all desktop devices including macOS Sonoma and Windows 11.
> 
> ## Is there anything else you think would be useful in recreating the issue?
> This is a core NHS style component; flagging this issue for your consideration

This issue should look at improving the contrast of the contents list dashed but also consider other similar components. For example the breadcrumbs and table components. 

## Why
Improving the contrast will enhance the legibility and therefore usability of some of our components that contain non-text elements with low contrast. Particularly for users with low vision.

## Other information to support your proposal
Related to:
* https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/25
* https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/6
* https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/26
* https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/36

**Comment by jason-lee-design** [comment]
In our accessibility audit, we've also had a similar non-compliance raised but about the arrows on the breadcrumb:

> The breadcrumb separator icon is a graphical object that helps users understand the content structure and navigation path, but it does not meet the minimum contrast requirement with a ratio of 1.84:1. This low contrast makes it difficult for users with low vision to perceive the icon and effectively navigate. To address this, adjust the icon's foreground or background colour to ensure a contrast ratio of at least 3:1, making it more visually distinct and accessible for all users.

**Comment by jason-lee-design** [comment]
Do we have an idea of when this might be fixed or are we best to apply an override just in the meantime? Thanks.

**Comment by andyscott4nhs** [comment]
Proposals sent to @anandamaryon1  - decision made, prototype to be built ready for dev

**Comment by anandamaryon1** [comment]
Done and released in [frontend 10.3.0](https://github.com/nhsuk/nhsuk-frontend/releases/tag/v10.3.0)
